app/models/grda_warehouse/tasks/client_cleanup.rb

```diff
@@ -10,6 +10,36 @@
 # also set the processed_hash on the enrollment to nil
 
 module GrdaWarehouse::Tasks
+  # The ClientCleanup task is a maintenance utility for ensuring the  integrity and
```

Comment:
@eanders - I generated some docs for this class while adding the hooks for CE. Look okay?